guides/source/action_mailbox_basics.md

```diff
@@ -374,9 +420,19 @@ class ForwardsMailboxTest < ActionMailbox::TestCase
     recording = people(:david).buckets.first.recordings.last
     assert_equal people(:david), recording.creator
     assert_equal "Status update?", recording.forward.subject
-    assert_match "What's the status?", recording.forward.content.to_s
+    assert_match "What's the status?", recording.forward.decoded.to_s
   end
 end
 ```
 
 Please refer to the [ActionMailbox::TestHelper API](https://api.rubyonrails.org/classes/ActionMailbox/TestHelper.html) for further test helper methods.
+
+## Incineration of InboundEmails
+
+By default, an `InboundEmail` that has been processed will be incinerated after 30 days. The InboundEmail is considered as processed when its status changes to `delivered`, `failed`, or `bounced`.
```

Comment:
This is the only instance where `InboudEmail` is not shown as code, let's wrap it too for consistency with the rest, since it's actually referring to the object in code:

```suggestion
By default, an `InboundEmail` that has been processed will be incinerated after 30 days. The `InboundEmail` is considered as processed when its status changes to `delivered`, `failed`, or `bounced`.
```

Alternatively, we can consider skipping the second mention in the paragraph:

```suggestion
By default, an `InboundEmail` that has been processed will be incinerated after 30 days. It is considered as processed when its status changes to `delivered`, `failed`, or `bounced`.
```